src/Gui/ViewProviderCoordinateSystem.cpp

```diff
@@ -168,6 +168,46 @@ bool ViewProviderCoordinateSystem::isTemporaryVisibility() {
     return !tempVisMap.empty();
 }
 
+void ViewProviderCoordinateSystem::setPlaneLabelVisibility(bool val)
+{
+    auto lcs = static_cast<App::LocalCoordinateSystem*>(getObject());
+    std::vector<App::Plane*> planes = lcs->planes();
+    for (auto* plane : planes) {
```

Comment:
I actually did this on purpose because I was under the impression that putting the variable in the for loop was calling planes() on every steps. So I took the habit of not doing that.

Reply:
This is however not the case, so it does not seem like a good habit.

Reply:
That's great then.